### PR TITLE
Adds FAQ page

### PR DESCRIPTION
**Pull request by schalkventer**

### CODE REVIEW

**Comment by schalkventer** [comment]
image

File: _data/2017-18/national/departments/agriculture-forestry-and-fisheries.yaml [hunk 1]
```diff
@@ -1,27 +1,27 @@
 contributed_datasets: null
 financial_years:
 - closest_match:
-    is_exact_match: true
+    is_exact_match: false
```
Comment (longhotsummer) [inline _data/2017-18/national/departments/agriculture-forestry-and-fisheries.yaml]: Are these changes expected?

**Comment by longhotsummer** [comment]
Please change the title to `Frequently Asked Questions (FAQ)`

**Comment by longhotsummer** [comment]
Please also add the links mentioned in the comments in the google doc.

File: _layouts/faq.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <p>The development of vulekamali is ongoing and additional data and resources are still being added to the site. Read more about the project timelines to learn about what types of datasets will be added, and when.</p>
+
+        <p>Please tell us about the data you’re looking for by emailing us at feedback@vulekamali.gov.za.</p>
+      </div>
+    </div>
+
+    <div class="Page-card">
+      <div class="Card">
+        <h2 class="Card-title">I have an idea for a feature or dataset to add to the vulekamali portal. Who should I send it to?</h2>
+        <p>lease send us your ideas by emailing them to feedback@vulekamali.gov.za.</p>
```
Comment (longhotsummer) [inline _layouts/faq.html]: Missing P

**Comment by schalkventer** [comment]
Right. Will change title, add links and fix issue with YAML.

**Comment by schalkventer** [comment]
image

@longhotsummer Changes made.

**Comment by longhotsummer** [comment]
Looks good.

**Pull request merged**